### DISCUSSION TITLE
How can I "full-size" a pane in a tab that has other panes?

### DISCUSSION THREAD

**Discussion by 1dancook** [wezterm/wezterm#5937] [Q&A] [answered]
I've been searching the docs but haven't seen anything. I use this in tmux (`resize-pane -Z`) and would love to know how I could do this as it's pretty essential for when I have "busy" workspaces.

**Comment by nekowinston** [comment, marked as answer]
Sounds like you're looking for [TogglePaneZoomState](https://wezfurlong.org/wezterm/config/lua/keyassignment/TogglePaneZoomState.html).

**Reply by 1dancook** [reply]
Fantastic! Thank you for the quick response!
